src/components/application_manager/include/application_manager/message_helper.h

```diff
@@ -217,20 +217,6 @@ class MessageHelper {
   static smart_objects::SmartObjectSPtr CreateSetAppIcon(
       const std::string& path_to_icon, uint32_t app_id);
 
-  DEPRECATED static bool SendIVISubscribtions(const uint32_t app_id,
```

Comment:
shall 
 typedef std::map<std::string, mobile_apis::VehicleDataType::eType> VehicleData;
also be moved to the new plugin?

Reply:
@yang1070 discussed during online meeting.